### ISSUE TITLE
Rename vkd3d to vkd3d-proton, and add version 2.5.

### ISSUE THREAD

**Issue by Bananaman** [bottlesdevs/components#92]
When I saw "vkd3d-v2.4" in the settings dialog, I was scared, thinking it's the basic VKD3D by the Wine project.

All development happens on VKD3D-Proton nowadays:

https://github.com/HansKristian-Work/vkd3d-proton/releases

It doesn't require Proton, it's just named that way because it's the Proton-project fork of VKD3D.

Well, I checked in the source files and was happy to see that you're actually using vkd3d-proton.

1. Would be best to rename it to vkd3d-proton in Bottles menu, to not scare people by making them think it's the upstream Wine version with limited functionality.
2. Version 2.5 is out so Bottles needs a small update of the installer menu to add this new version: https://github.com/HansKristian-Work/vkd3d-proton

:)

**Comment by Bananaman** [comment]
I was looking into making a pull request for the update but I dunno. I see this file:

https://github.com/bottlesdevs/components/tree/main/vkd3d

Which reveals that you pull from a Lutris fork of VKD3D-Proton:

https://github.com/lutris/vkd3d/

This seems really strange to me. It's hundreds of commits behind the real project?

Let's just pull directly from the real project at https://github.com/HansKristian-Work/vkd3d-proton/releases.

Because this is just a DLL file which emulates DX12 and translates it to Vulkan graphics API. It isn't dependent on Proton or Steam. So we should go straight to Valve's official version at the HansKristian-Work repo. That's where the innovation happens.

**Comment by mirkobrombin** [comment]
Official vkd3d builds came in zst format which is not supported in Bottles. Lutris provide builds in tar.xz format which is compatible. We are planning to build our vanilla builds, until this we stuck on lutris one

**Comment by Bananaman** [comment]
@mirkobrombin Ah okay I understand, makes sense, I see the `.zst` extension on the releases yeah.

Well, it looks like Lutris fork has 2.5 too, so we can update to that one for now. :)

https://github.com/lutris/vkd3d/releases

**Comment by mirkobrombin** [comment]
> 

The problem with lutris releases is that they eventually remove them or update attachments, I am waiting to see if this is not happening to update. We will probably finish our builds in the meantime

**Comment by Bananaman** [comment]
@mirkobrombin Ahh I see, yeah that's a problem. Also it's weird that their 2.5 build is on an older tag than the official 2.5 release.

Well, I can do without DXR raytracing support in the meantime. It's not very useful for most games. :D

**Comment by Bananaman** [comment]
@mirkobrombin Hey :) I was just working on trying to add VKD3D 2.5 and I noticed this commit:

https://github.com/bottlesdevs/components/commit/8d567b3185b4623c921f539ff0fd964f074aebc0

One surprising thing is that v2.5 has renamed the DLL folders.

- `setup_vkd3d_proton.sh`: Identical in 2.4 and 2.5
- 32-bit folder: `x32` in 2.4, `x86` in 2.5
- 64-bit folder: `x64` in both.

**Comment by mirkobrombin** [comment]
Bottles handle dxvk and vkd3d with their official scripts. So every path change do not compromise the components installation.

**Comment by Bananaman** [comment]
@mirkobrombin Yeah I looked in `vkd3d-proton-2.5/setup_vkd3d_proton.sh`, at the top you will see:

```
vkd3d_lib32=${vkd3d_lib32:-"x86"}
vkd3d_lib64=${vkd3d_lib64:-"x64"}
```

It loads the DLLs from the correct dirs. :)

**Comment by mirkobrombin** [comment]
Closed by 6e0c51fed6bcb37d36d8f37dff563f5b4ea940db

**Comment by Bananaman** [comment]
@mirkobrombin Great news! Thanks! <3
